_posts/2018-11-28-how-we-work-exercise-and-stand-desk.md

```diff
@@ -0,0 +1,83 @@
+we increased our work completed by 17% for a 4-month span. We have also encouraged other ITProTV
+employees to join our Slack channel and together we keep each other accountable in living a healthy
+lifestyle.
+
+## Stand Desk
+
+For our engineering team, stand desks offer another great way to keep a healthy lifestyle. As a team,
+our use of the stand desk varies. I stand a few times a day and generally use a balance
+board, while one of my coworkers stands a majority of the day. If you've read our previous
+[post]({{ site.baseurl }}{% post_url 2018-10-24-how-we-work-pair-programming %}) on
+pair programming, you know that we spend a lot of time at stand desk pairing stations, so we tend
+to agree to stand or sit for a given pairing rotation.
+
+As far as research is concerned, stand desk allow your body to burn a [modest][] amount of calories,
+but for us, it is so much more. The biggest benefits for our team are:
+
+1. Less stiff and stagnant
+1. Better posture
+1. Being prepared to fight any ninjas who may come into our space!
+
+These benefits really help us stay healthy and get moving as a team. Together with exercising, our
+morale stays positive and we continue to see our productivity increase.
+
+## How To Get Started
+
+Implementing these two ideas for your workspace will most likely vary in difficulty. For exercise, there is no
+real cost associated. All you really need to do is get your coworkers on board and pick a time of the hour that
+works best for each of you. I would recommend getting your whole team to do it, but if that's not possible, have
+at least one other person do it with you. The accountability alone will increase your chances of success.
+
+Getting stand desks may be a little bit more challenging because there is a cost associated. The cost of stand
+desks range from $400 to $1000+, but fear not, there are other alternatives, such as a [cardboard][] stand desk
+or an [adjustable height desktop][] stand desk. If you aren't ready to fully commit, I would definitely recommend
+a cheaper option to see if a stand desk works for you.
+
+As a team, we really only had one hesitation when beginning to exercise and that was the interruption of flow, but
+after months of this experiment, exercise is here to stay in our daily routine, so I would encourage you to take a
+leap and see if it works for you!
+
+## In Review
+
+Over the last 5 months, our team has taking the time to invest in our health by exercising
+every hour and utilizing our stand desks. The results speak for themselves. Our moral is higher
+and our productivity improved. Our tactics may not work for you, but I would encourage you to
+take step towards implementing some of these ideas. I don't think you'll regret it!
+
+
+[A photo of ITProTV Team doing push ups]: /assets/2018-11-26-push-ups.jpg
+[modest]: https://journals.sagepub.com/doi/full/10.1177/2047487317752186
+[cardboard]:https://www.amzn.com/B01GGNV6R0
```

Comment:
Does this work without a space after the first colon?

Reply:
It does work, but I'll fix it for consistencies sake :)